<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::ActivityTypeManager</class>
 <widget class="QWidget" name="tt3::gui::ActivityTypeManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="activityTypesTreeWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="buttonsWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="modifyActivityTypePushButton">
        <property name="text">
         <string>Modify activity type</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/ModifyActivityTypeSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/ModifyActivityTypeSmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="createActivityTypePushButton">
        <property name="text">
         <string>Create activity type</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/CreateActivityTypeSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/CreateActivityTypeSmall.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="destroyActivityTypePushButton">
        <property name="text">
         <string>Destroy activity type</string>
        </property>
        <property name="icon">
         <iconset resource="tt3-gui.qrc">
          <normaloff>:/tt3-gui/Resources/Images/Actions/DestroyActivityTypeSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/DestroyActivityTypeSmall.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="filterWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="filterLabel">
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="filterLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tt3-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>activityTypesTreeWidget</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>tt3::gui::ActivityTypeManager</receiver>
   <slot>_activityTypesTreeWidgetCurrentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activityTypesTreeWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>tt3::gui::ActivityTypeManager</receiver>
   <slot>_activityTypesTreeWidgetCustomContextMenuRequested(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createActivityTypePushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::ActivityTypeManager</receiver>
   <slot>_createActivityTypePushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modifyActivityTypePushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::ActivityTypeManager</receiver>
   <slot>_modifyActivityTypePushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>destroyActivityTypePushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::ActivityTypeManager</receiver>
   <slot>_destroyActivityTypePushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tt3::gui::ActivityTypeManager</receiver>
   <slot>_filterLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_activityTypesTreeWidgetCurrentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>_activityTypesTreeWidgetCustomContextMenuRequested(QPoint)</slot>
  <slot>_createActivityTypePushButtonClicked()</slot>
  <slot>_modifyActivityTypePushButtonClicked()</slot>
  <slot>_destroyActivityTypePushButtonClicked()</slot>
  <slot>_filterLineEditTextChanged(QString)</slot>
 </slots>
</ui>
